packages/types/src/clerk.ts

```diff
@@ -89,6 +89,9 @@ export interface Clerk {
   /** Current User. */
   user?: UserResource | null;
 
+  /** Current state hash derived from userId, sessionId and orgId */
+  authStateKey?: string;
```

Comment:
As we've discussed, we'll be moving our FE SDK docs inline using JSDoc. In preparation for that, let's treat this JSDoc comment as the documentation for this prop and provide a better/fuller description and an example (eg This is especially useful if you want to invalidate caches (eg: for react-quiery, useSwr etc) . . .`

Also, let's stick with the 
```
/**
*
*/
```
even for single-line comments from now on